lib/bolt/config/transport/ssh.rb

```diff
@@ -12,94 +12,132 @@ class SSH < Base
-                                          "`sudo-password`, since `run-as-command` does not use `sudo-password` "\
-                                          "or support prompting. The command executed on the target is "\
-                                          "`<sudo-executable> -S -u <user> -p custom_bolt_prompt <command>`. "\
-                                          "**This option is experimental.**" },
-          "sudo-password"      => { type: String,
-                                    external: true,
-                                    desc: "Password to use when changing users via `run-as`." },
-          "tmpdir"             => { type: String,
-                                    external: true,
-                                    desc: "The directory to upload and execute temporary files on the target." },
-          "tty"                => { type: TrueClass,
-                                    desc: "Request a pseudo tty for the session. This option is generally "\
-                                          "only used in conjunction with the `run-as` option when the sudoers "\
-                                          "policy requires a `tty`." },
-          "user"               => { type: String,
-                                    external: true,
-                                    desc: "Login user." }
+          "cleanup"               => { type: TrueClass,
+                                       external: true,
+                                       desc: "Whether to clean up temporary files created on targets." },
+          "connect-timeout"       => { type: Integer,
+                                       desc: "How long to wait when establishing connections." },
+          "copy-command"          => { external: true,
+                                       desc: "Command to use when copying files using ssh-command. "\
+                                             "Bolt runs `<copy-command> <src> <dest>`. **This option is "\
+                                             "experimental.**" },
+          "disconnect-timeout"    => { type: Integer,
+                                       desc: "How long to wait before force-closing a connection." },
+          "encryption-algorithms" => { type: Array,
+                                       desc: "List of encryption algorithms to support when establishing a "\
+                                             "connection with a target. Supported algorithms are "\
+                                             "defined by the Ruby net-ssh library and can be viewed "\
+                                             "[here](https://github.com/net-ssh/net-ssh#supported-algorithms). "\
+                                             "All supported, non-deprecated algorithms are enabled by default when "\
+                                             "this option is not used. To enable support for default algorithms "\
```

Comment:
```suggestion
                                             "this option is not used. To use all default algorithms "\
```

Reply:
'reference'? 'specify'? might also work. Paging @hestonhoffman

Reply:
Honestly overall this message is pretty good at communicating a weird situation. There's like...3 possible states for any algorithm to be in *in the net-ssh lib alone*, and then that's on top of whatever the user has configured locally.